### PR TITLE
use SlotComposer to render suggestions instead of AppShell

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: dev/app-shell/elements/arc-host.html [hunk 1]
```diff
@@ -69,10 +72,13 @@
       let context = await this._loadManifest(config, loader);
       // composer
       let slotComposer = new Arcs.SlotComposer({
-        rootContext: this,
+        rootContext: this.parentElement,
         affordance: config.affordance,
-        containerKind: config.containerKind
+        containerKind: config.containerKind,
+        suggestionsContext: config.suggestionsNode
```
Comment (mariakleiner) [inline dev/app-shell/elements/arc-host.html]: in slot-composer we initialiaze _suggestionContext to:
this._suggestionsContext = options.suggestionsContext || this._contextById["suggestions"];

so given that this PR already sets a slot with "suggestions" ID, do we really need to set suggestionContext in |options| too?

**Review by mariakleiner — APPROVED** [review]

File: dev/app-shell/elements/arc-host.html (hunk 1 above)
Comment (sjmiles) [reply dev/app-shell/elements/arc-host.html]: Just thought I would leave that door open, could go either way.

Comment (mariakleiner) [reply dev/app-shell/elements/arc-host.html]: maybe then just comment on it, so that we remember to remove it later? not critical though.

Comment (sjmiles) [reply dev/app-shell/elements/arc-host.html]: This is also supported here, fwiw: https://github.com/PolymerLabs/arcs/pull/611/files#diff-a444146ecefc6f8971a371551f85cf0aR34

**Pull request merged**